lib/mspec/helpers/numeric.rb

```diff
@@ -15,11 +16,13 @@ def bignum_value(plus = 0)
 end
 
 def max_long
-  2**(0.size * 8 - 1) - 1
+  long_byte_size = [0].pack('l!').size
```

Comment:
We should use `PlatformGuard::C_LONG_SIZE` here and below to avoid recomputing it every single time.

Reply:
I'll do it

Reply:
https://github.com/ruby/mspec/commit/6499325c5fc4044f26a41ee3a769a67d79062229

Reply:
Good catch.